themes/bootstrap3/templates/search/results.phtml

```diff
@@ -118,3 +118,13 @@
   <?php endforeach; ?>
 </div>
 <?php /* End Refine Search Options */ ?>
+<?php
+  // Set up the page title now that we know the number of results
+
+  // $this->layout()->srmessage was initialized when showing.phtml was rendered,
+  // and it might have been altered if results were empty. However, it contains
+  // HTML tags like <strong> as well as encoded entities that we need to tidy.
+  $headTitle = html_entity_decode(strip_tags($this->layout()->srmessage), ENT_QUOTES | ENT_HTML5, 'UTF-8');
+  $headTitle = $this->slot('head-title')->get($headTitle);
```

Comment:
Was this meant to be `->set($headTitle)`?

Reply:
I think it's meant to be get, because we that allows a higher-level template to override the settings here. We're just providing a default if no higher-level code overrode it.

Reply:
(If I'm remembering correctly, this functionality is used by some of the specialized search pages like course reserves, new items, etc.).

Reply:
Yes! Thank you! I was confused by my own functionality!